hack/make/.ensure-httpserver

```diff
@@ -8,7 +8,7 @@ dir="$DEST/httpserver"
 mkdir -p "$dir"
 (
 	cd "$dir"
-	GOOS=linux GOARCH=amd64 go build -o httpserver github.com/docker/docker/contrib/httpserver
+	go build -o httpserver github.com/docker/docker/contrib/httpserver
```

Comment:
We will fix this change as it breaks the Windows-PRs build compiling a windows httpserver binary and building a Linux container with it.

We weren't sure if this is still needed, but already planned for this situation to have the right values for GOOS and GOARCH of the Docker Engine used in the integration tests.